lib/wcc/contentful/store.rb

```diff
@@ -44,20 +44,23 @@ def validate!
       end
 
       def build_eager_sync(config, store = nil, *_options)
+        store = nil if [{ preview: false }, { preview: true }].include?(store)
```

Comment:
I don't get this - when does this occur?

Reply:
I'm sure you have a better way to solve this. I was just trying to move quickly, and once I realized why the tests were failing, I just wanted to fix it without breaking anything else, but I'm not sure if I actually accomplished that or not.